metadata-ingestion/source_docs/bigquery.md

```diff
@@ -1,14 +1,17 @@
 # BigQuery
 
+To get all metadata from BigQuery you need to use two plugins `bigquery` and `bigquery-usage`. Both of them are described in this page. These will require 2 separate recipes. We understand this is not ideal and we plan to fix this problem in future to have a single plugin.
```

Comment:
I would avoid wording as "fix this problem" and instead say we intend to "make this easier in the future". Don't lock into how either.

Reply:
fixed